packages/react-events/src/__tests__/Focus-test.internal.js

```diff
@@ -258,17 +261,76 @@ describe('Focus event responder', () => {
       expect(onFocusChange).toHaveBeenCalledTimes(2);
       expect(onFocusChange).toHaveBeenCalledWith(false);
     });
+
+    it('is not called after "blur" and "focus" events on nested descendants', () => {
```

Comment:
This is incorrect behaviour

Reply:
Why? It is verifying that `onFocusChange` is not called when focusing a descendant rather than the immediate child of the `<Focus>` component?

Reply:
Oh I missed that it wasn't starting with the root being focused. Given the UX is quite broken by the patch it's odd that no tests failed.

Reply:
Yeah, I'll look into it and add more tests.